### ISSUE TITLE
SaveAction -> Format Source Code

### ISSUE THREAD

**Issue by spuckydaslama** [peichhorn/lombok-pg#35]
Formatting the source code as part of saving in eclipse (SaveAction) seems not to be working anymore, if lombok did his magic in a class. Organizing the imports still works btw and also, manually formatting(ctrl+f) works as well.

the following sample does not formatting anyhting on saving

```
import lombok.AllArgsConstructor;

@AllArgsConstructor
public class NoEclipseAutoFormatting {
    private void test() {
        int x     =        1   +   2  ;//i just pressed ctrl+s
    }
}
```

.. without lombok in a similar class ..

```
public class NoLombokAndFormattingWorks {
    private void test() {
        int x = 1 + 2;//i just pressave ctrl+s
    }
}
```

**Comment by peichhorn** [comment]
Yeah this is a known issue of lombok and pretty hard to fix, see [lomboks issue tracker](http://code.google.com/p/projectlombok/issues/list) for further information. You should even experience that some refactorings are not working on lombok annotated methods. On the plus side once you got rid of all the boilerplate, there is usually not that much left to refactor.

I leave this issue open, maybe it helps me remembering, that some advanced eclipse features of lombok still need some work.

**Comment by peichhorn** [comment]
fixed with the upgrade to lombok 0.10.3 including the usability fixes provided by Jappe van der Hel
see issue #49

I will let you now when I upload a new edge release.

**Comment by peichhorn** [comment]
Please try out the new [edge release](http://cloud.github.com/downloads/peichhorn/lombok-pg/lombok-pg-0.10.2-EDGE.jar).
